module_utils/network/argspec/base.py

```diff
@@ -0,0 +1,4 @@
+class ArgspecBase(object):
```

Comment:
What is the purpose of this class?

Reply:
This is empty as of now. When we merge this POC with code generator, we can decide then?